src/mame/namco/namcos2_m.cpp

```diff
@@ -264,7 +266,7 @@ uint16_t namcos2_state::namcos2_68k_key_r(offs_t offset)
 		case 1: return 0x110;
 		case 4: return 0xBE;
 		case 6: return 0x1001;
-		case 7: return (m_sendval==1)?0xBE:1;
+		case 7: return (m_sendval == true) ? 0xBE : 1;
```

Comment:
You don’t need to test Booleans with `== true`, you can just do `m_sendval ? 0xbe : 1`